src/www/widgets/widgets/ipsec.widget.php

```diff
@@ -134,15 +146,18 @@
         <td><?= (count($ipsec_tunnels) - $activetunnels); ?></td>
         <td>
 <?php
-        // count active mobile users
-        $mobile_users = 0;
-        foreach ($ipsec_leases as $lease) {
-            if ($lease['online']) {
-                ++$mobile_users;
-            }
+    // Initialize variable unique_users
+    $unique_users = array();
+    foreach ($ipsec_leases as $lease) {
+        if ($lease['online']) {
```

Comment:
I would push this one in the other loop, so you only have to travers `$ipsec_leases` once.